src/components/OpenSeadragonViewer.js

```diff
@@ -11,6 +11,31 @@ import CanvasWorld from '../lib/CanvasWorld';
  * and rendering OSD.
  */
 export class OpenSeadragonViewer extends Component {
+  /**
```

Comment:
Did this method get moved around in the class? Was this due to a linting suggestion?

Reply:
Yeah, since we're passing `currentAnnotations` as a parameter to the function (instead of using the annotations props) so that it can be used for different types (selected vs. highlighted) of annotations it's no longer using `this` so it needs to be a `static` function (which apparently are required to come before the constructor).

Reply:
oh wow.. got it thanks.